bundle/src/main/java/com/adobe/acs/commons/mcp/util/AnnotatedFieldDeserializer.java

```diff
@@ -84,7 +86,13 @@ private static void parseInput(Object target, ValueMap input, Field field) throw
             if (value.getClass().isArray()) {
                 val = ((Object[]) value)[0];
             }
-            parseInputValue(target, String.valueOf(val), field);
+
+            if (val instanceof InputStream) {
```

Comment:
@badvision Changed this to handle file uploads.

Reply:
Makes sense.